package.json

```diff
@@ -89,6 +90,7 @@
     "release-it": "^15.3.0",
     "release-it-lerna-changelog": "^5.0.0",
     "replace": "^1.2.1",
+    "resize-observer-polyfill": "^1.5.1",
```

Comment:
@theodoreb why was this added? If it is required, does it need to be in `dependencies` instead of `devDependencies`?

Reply:
it's only for running the tests because floatingui `autoUpdate` uses ResizeObserver and that fails if it's not available.

Reply:
see https://floating-ui.com/docs/autoUpdate